HerPublicWebsite/Services/QuestionnaireService.cs

```diff
@@ -175,7 +175,10 @@ public async Task<Questionnaire> RecordNotificationConsentAsync(bool consentGran
 
     public void SaveQuestionnaireToSession(Questionnaire questionnaire)
     {
+        var oldQuestionnaireString = httpContextAccessor.HttpContext!.Session.GetString(SessionKeyQuestionnaire);
+        if (oldQuestionnaireString == null) sessionRecorderService.RecordNewSessionStarted();

```

Comment:
Can we make this a suitably-named boolean instead? Think it would be clearer what we're doing (otherwise "what is an old questionnaire?")

Reply:
sure